src/show/pages/public/CreateProject/index.js

```diff
@@ -0,0 +1,81 @@
+import axios from 'axios';
+import { useState } from 'react';
+import StepWizard from 'react-step-wizard';
+import { useNavigate } from 'react-router-dom';
+import { PROJECT_PATH } from '../../../../process/routes/paths';
+
+import {
+  FeeStep,
+  GoalStep,
+  TitleStep,
+  ImagesStep,
+  PublishStep,
+  CategoryStep,
+  DatelineStep,
+  StatementStep,
+  DescriptionStep,
+  TeamMembersStep
```

Comment:
I know it looks good with them sorted by length, but in there they are sorted abc.

Reply:
now it looks like a christmas tree without a top :(